tox.ini

```diff
@@ -113,12 +113,14 @@ passenv =
     INDEX_URL_BUILD_ARG
 changedir=k8s_itests/
 commands =
-    # Build /etc/paasta used by docker-compose
-	{toxinidir}/k8s_itests/scripts/setup.sh
+    # Fail fast if Docker Compose V2 is not available
+    docker compose version
```

Comment:
huh, this doesn't work on our devbox (at least, for me)

Reply:
I'm checking the jenkins run tho and its passing: https://github.com/Yelp/paasta/actions/runs/15856531471/job/44703167701?pr=4065

Reply:
erroring locally, looking into it